integrate-kobiton-travisci.md

```diff
@@ -0,0 +1,190 @@
+# Run Kobiton Automation Test on Travis CI
```

Comment:
As a developer, I don't want to read entire ~200 lines of documentation to do this, I imagine for users to actually archive the goal as this guideline indicated: `Execute automation tests on Kobiton devices with Travis CI`, they just have to do 3 steps:
- Prepare the prerequisites
- Clone the code/configurations in this repo
- Add their credential information

Therefore, adding example code/configuration to this is essential as users will easier to reason about our docs & we don't need to make it too verbose like this.

Reply:
I think in different ways, the target of this article should more general and the way you suggest is much more in technical.